src/main/scala/com/fulcrumgenomics/umi/ReviewConsensusVariants.scala

```diff
@@ -160,6 +160,13 @@ class ReviewConsensusVariants
   private val refFile = ReferenceSequenceFileFactory.getReferenceSequenceFile(ref)
   private val dict = refFile.getSequenceDictionary
 
+  validate(this.refFile.isIndexed,
+    f"The reference file has not indexed (use `samtools faidx ref.fa`): ${this.ref}"
+  )
+  validate(this.refFile.getSequenceDictionary != null,
+    f"The reference file has no sequence dictionary (use `samtools dict ref.fa > ref.dict`): ${this.ref}"
+  )
```

Comment:
Similarly, something like the following would be nice:

```suggestion
  validate(this.refFile.getSequenceDictionary != null,
    f"The reference file has no sequence dictionary (use `samtools dict ${this.ref} -o  ${PathUtil.replaceExtension(this.ref, ".dict")}`)"
  )
```